draft-ietf-ppm-dap.md

```diff
@@ -1722,13 +1750,17 @@ preparation continuation messages constructed in the previous step. The
 The Leader MUST authenticate its requests to the Helper using a scheme that
 meets the requirements in {{request-authentication}}.
 
-The Helper's response will be an `AggregationJobResp` message (see
-{{aggregation-helper-init}}). The response's `prepare_resps` must include
-exactly the same report IDs in the same order as the Leader's
-`AggregationJobContinueReq`. Otherwise, the Leader MUST abort the aggregation
-job.
+After receiving HTTP status 202 Accepted, the Leader proceeds to poll the
+aggregation job by sending requests to
+`GET {helper}/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}?step={step}`
+where `step` is the `step` field of the `AggregationJobContinueReq`. It polls
+until receiving HTTP status 200 OK with a body containing an
+`AggregationJobResp`. The Helper's response SHOULD include a Retry-After header
+field to suggest a polling interval to the Leader.
```

Comment:
As above, this suggests the 200 OK should contain `Retry-After`. This sentence should be attached to the text describing the 202 Accepted response.